tests/interfaces/test_jax_jit.py

```diff
@@ -137,13 +137,15 @@ def cost(a):
 
         # adjoint method only performs a single device execution, but gets both result and gradient
```

Comment:
Does this comment still apply? :thinking:

Reply:
updated.